portfolio/src/main/webapp/script.js

```diff
@@ -33,76 +33,120 @@ function addRandomFact() {
 }
 
 /**
- * Cycles backwards to the previous background image
+ * Cycles backwards to the previous background image.
  */
 function prevBackground() {
   changeBackground(-1);
 }
 
 /**
- * Cycles forwards to the next background image
+ * Cycles forwards to the next background image.
  */
 function nextBackground() {
   changeBackground(1);
 }
 
-let curBackground = 0;
+let currentBackgroundIndex = 0;
+
+// List of background images and their color themes.
+const backgroundImgs = [
```

Comment:
nit: maybe backgroundThemes (since the config below includes more than images). Otherwise spell out images (https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers)